src/System.IO.Ports/tests/Legacy/SerialPort/DosDevices.cs

```diff
@@ -16,27 +17,28 @@ public DosDevices()
         Initialize();
     }
 
+/*
```

Comment:
I guess because it looked a bit different to all the other 100-or-so main() methods I'd deleted in each of the other test classes.   It's just debugging help as far as I can see - shall we just get rid of it?